OutOfSchool/OutOfSchool.WebApi.Tests/Extensions/MapSearchResultToOkOrNoContentExtensionTests.cs

```diff
@@ -0,0 +1,90 @@
+﻿using System.Collections.Generic;
+using FluentAssertions;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using NUnit.Framework;
+using OutOfSchool.BusinessLogic.Models;
+using OutOfSchool.BusinessLogic.Models.Application;
+using OutOfSchool.WebApi.Extensions;
+
+namespace OutOfSchool.WebApi.Tests.Extensions;
+
+[TestFixture]
+public class MapSearchResultToOkOrNoContentExtensionTests
+{
+    private class TestController : ControllerBase
+    {
+    }
```

Comment:
(Excessive class) `TestController` don't change `ControllerBase`, thus latter could be used, and `TestController` is not needed. 
(Naming) JFYI, as class will be deleted: This test checks extension, not the controller. Thus controller is needed as parameter for the extension method. Thus it should be named not `TestController` but `Fake...` or `Mock...`

Reply:
I was thinking, that i can not create a mock of ControllerBase, because it is abstract (you can not create instance of  abstract class - you can not create a mock of it). My bad.

Reply:
Fixed.